.github/workflows/build.yml

```
@@ -0,0 +1,94 @@
+      version_major: ${{ steps.build_info.outputs.version_major }}
+      version_minor: ${{ steps.build_info.outputs.version_minor }}
+      version_patch: ${{ steps.build_info.outputs.version_patch }}
+      image_tag: ${{ steps.build_info.outputs.image_tag }}
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - id: build_info
+      name: Declare build info
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:6:1: branch appears unused. Verify use (or export if used externally)

(shellcheck)